angular/src/lib/checkbox/checkbox.component.ts

```diff
@@ -24,35 +25,36 @@ const CUSTOM_CHECKBOX_CONTROL_VALUE_ACCESSOR: any = {
 @Component({
   selector: 'md-checkbox',
   template: `
-    <div class="md-input-group md-checkbox" [ngClass]="wrapperClasses">
-      <input
-        class="md-input md-checkbox__input"
-        [ngClass]="inputClasses"
-        type="checkbox"
-        (change)="handleChange($event)"
-        [value]="value"
-        [checked]="checked"
-        [name]="name"
-        [attr.id]="htmlId"
-        [attr.tabindex]="tabIndex"
-        [disabled]="disabled"
-        [required]="required"
-      />
-
-      <label
-        *ngIf="label"
-        class="md-checkbox__label"
-        [attr.for]="htmlId"
-        (click)="onClick($event)"
-      >
-        <span>{{ label }}</span>
-      </label>
-
-      <ng-content></ng-content>
-    </div>
+    <input
+      class="md-input md-checkbox__input"
+      [ngClass]="{'indeterminate': indeterminate}"
+      type="checkbox"
+      (change)="handleChange($event)"
+      [value]="value"
+      [checked]="checked"
+      [name]="name"
+      [attr.id]="htmlId"
+      [attr.tabindex]="tabIndex"
+      [disabled]="disabled"
+      [required]="required"
+    />
+
+    <label
+      *ngIf="label"
+      class="md-checkbox__label"
+      [attr.for]="htmlId"
+      (click)="onClick($event)"
+    >
+      <span>{{ label }}</span>
+    </label>
+
+    <ng-content></ng-content>
   `,
   styles: [],
   providers: [CUSTOM_CHECKBOX_CONTROL_VALUE_ACCESSOR],
+  host: {
+    class: 'md-input-group md-checkbox',
```

Comment:
Change to className input

Reply:
adding to input ad user can add class to host